Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -419,6 +419,7 @@ const Info<bool> MAIN_USE_HIGH_CONTRAST_TOOLTIPS{
 const Info<bool> MAIN_USE_PANIC_HANDLERS{{System::Main, "Interface", "UsePanicHandlers"}, true};
 const Info<bool> MAIN_ABORT_ON_PANIC_ALERT{{System::Main, "Interface", "AbortOnPanicAlert"}, false};
 const Info<bool> MAIN_OSD_MESSAGES{{System::Main, "Interface", "OnScreenDisplayMessages"}, true};
+const Info<int> MAIN_OSD_FONT_SIZE{{System::GFX, "Settings", "OSDFontSize"}, 13};
```

Comment:
Pull in default if possible?

Reply:
This could be where the default value is pulled from. Otherwise it would need a new #include, and 'm not sure what would be appropriate.